### PR TITLE
Solves the issue 932

### PR DESCRIPTION
**Pull request by gh640**
I added the logic to check if the `settings.FILE_CHARSET` exists or not before accessing the value.

### CODE REVIEW

File: compressor/filters/base.py [hunk 1]
```diff
@@ -119,7 +119,9 @@ class CompilerFilter(FilterBase):
     """
     command = None
     options = ()
-    default_encoding = settings.FILE_CHARSET
+    default_encoding = (
+        settings.FILE_CHARSET if hasattr(settings, 'FILE_CHARSET') else 'utf-8'
```
Comment (karyon) [inline compressor/filters/base.py]: does that work? i just fired up a console and the `hasattr` returns true on django 2.2. You could experiment with  `settings.is_overridden`, perhaps `settings.FILE_CHARSET if settings.is_overridden('FILE_CHARSET') else 'utf-8'` might work?

File: compressor/tests/test_filters.py [hunk 2]
```diff
@@ -31,16 +31,20 @@ def blankdict(*args, **kwargs):
 
 @override_settings(COMPRESS_CACHEABLE_PRECOMPILERS=('text/css',))
 class PrecompilerTestCase(TestCase):
+    CHARSET = (
+        settings.FILE_CHARSET if hasattr(settings, 'FILE_CHARSET') else 'utf-8'
+    )
```
Comment (karyon) [inline compressor/tests/test_filters.py]: i think for the tests (here and in that other file), we can indeed just replace every FILE_CHARSET with utf-8. i mean we're reading our own content here, and that should be utf-8...

File: compressor/filters/base.py (hunk 1 above)
Comment (gh640) [reply compressor/filters/base.py]: Thanks for the review. You're right. The check with `hasattr()` didn't work properly and I replaced it with `is_overridden()` with the new commit. It seems to work well now.

File: compressor/tests/test_filters.py (hunk 2 above)
Comment (gh640) [reply compressor/tests/test_filters.py]: I also replaced them to just `'utf-8'` in the new commit.

**Comment by gh640** [comment]
@karyon thanks very much. I fixed the points and I'd like you to review this again.

**Comment by karyon** [comment]
cool, thanks!

**Comment by gh640** [comment]
Thanks!

**Pull request merged**